package.json

```diff
@@ -15,7 +15,6 @@
     "@rails/webpacker": "^5.2.2",
     "axios": "^0.26.1",
     "bootstrap": "^4.6.0",
-    "datatable-sorting-datetime-moment": "^1.0.1",
```

Comment:
you have removed the `require('moment');` next to `require('datatable-sorting-datetime-moment');` in application.js, but only removed the second one here. how come? `moment` can be removed entirely here as well, or am i wrong?

Reply:
52173b4

Reply:
you are right. i initially wanted to remove it but wasn't sure, so I left it